ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-magic-numbers */
+import { expect, test } from '@playwright/test';
+import {
+  waitForIndigoToLoad,
+  selectNestedTool,
+  SelectTool,
+  selectTool,
+  LeftPanelButton,
+  clickInTheMiddleOfTheScreen,
+} from '@utils';
+
+test.describe('Hot keys', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
```

Comment:
Could you, please use `waitForPageInit`?

Reply:
If you use `waitForPageInit`, there is no need to use `await page.goto('')` as it is already inside that function